.github/workflows/main.yml

```diff
@@ -111,3 +111,28 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: ghcr.io/dunst-project/docker-images:misc-doxygen
+
+  appimage:
+    runs-on: ubuntu-22.04
```

Comment:
The reason behind ubuntu-22.04 is that on 24.04 (latest) a lot of CI broke due to ubuntu disabling access to user namespaces, now in this case it shouldn't hurt since nothing here is using that I think.

Reply:
See [here](https://github.com/ivan-hc/OBS-Studio-appimage/pull/14/files) where an appimage workflow that uses an archlinux container stopped working due to these [changes](https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces)

WIth that said, I haven't actually tired here to see if it fails or not.